DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: nestcolor
 Title: Colors for NEST graphs
-Version: 0.0.0.9013
+Version: 0.3.10
```

Comment:
quite a big step up :D

Reply:
This is a test from @arkadiuszbeer 
You can ignore this